include/mc_rtc/Configuration.h

```diff
@@ -278,6 +278,13 @@ struct MC_RTC_UTILS_DLLAPI Configuration
    */
   operator Eigen::Vector3d() const;
 
+  /*! \brief Retrieve as a Eigen::Vector4d instance
+   *
+   * \throws If the underlying value does not hold a numeric
+   * sequence of size 4
+   */
+  operator Eigen::Vector4d() const;
+
```

Comment:
I guess this is required to pass the colors as `Eigen::Vector4d`?

If we really want to add this we should also have:
- `add(const std::string &, const Eigen::Vector4d &)`
- `push(const Eigen::Vector4d &)`
- corresponding tests in `tests/testConfiguration.cpp`

Reply:
I used `mc_rtc::gui::Color` instead, but regardless I implemented all functions to properly handle `Eigen::Vector4d`. This might come in handy at some point ;)